core/jobs/batch_jobs/manual_voice_artist_name_job.py

```diff
@@ -181,69 +189,78 @@ def get_content_id_mapping_and_voiceovers_count(
             nested dictionary maps language codes to voiceover dicts that are
             present in the exploration.
             - The number of existing voiceovers in the exploration.
+
+        Raises:
+            Exception. Failed to get voiceover mapping for the given
+                exploration.
         """
         number_of_voiceovers: int = 0
         content_id_to_voiceover_mapping: Dict[str, Dict[
-            str, voiceover_models.VoiceoverDict]] = (
+            str, state_domain.VoiceoverDict]] = (
                 collections.defaultdict(dict))
+        try:
```

Comment:
Same as above, why are we doing a big try/except here? If it's justified, then it needs at least a docstring explaining, ditto above.